ZKSyncSDK/Classes/Wallet/Wallet+PromiseInterface.swift

```diff
@@ -14,17 +14,17 @@ public extension Wallet {
         return Promise { self.getAccountState(completion: $0.resolve) }
     }
     
-    func getTransactionFeePromise(for transactionType: TransactionType, tokenIdentifier: String) -> Promise<TransactionFeeDetails> {
-        return Promise { self.getTransactionFee(for: transactionType, tokenIdentifier: tokenIdentifier, completion: $0.resolve) }
-    }
-    
-    func getTransactionFeePromise(for transactionType: TransactionType, address: String, tokenIdentifier: String) -> Promise<TransactionFeeDetails> {
-        return Promise { self.getTransactionFee(for: transactionType, address: address, tokenIdentifier: tokenIdentifier, completion: $0.resolve) }
-    }
-    
-    func getTransactionFeePromise(for batchRequest: TransactionFeeBatchRequest) -> Promise<TransactionFeeDetails> {
-        return Promise { self.getTransactionFee(for: batchRequest, completion: $0.resolve) }
-    }
+//    func getTransactionFeePromise(for transactionType: TransactionType, tokenIdentifier: String) -> Promise<TransactionFeeDetails> {
+//        return Promise { self.getTransactionFee(for: transactionType, tokenIdentifier: tokenIdentifier, completion: $0.resolve) }
```

Comment:
Small nitpick: It would be better if the codebase didn't have any commented out code (or at least huge chunks of it)

Reply:
Sorry, it was outdated